qemu/tests/qemu_guest_agent.py

```diff
@@ -3933,13 +3933,25 @@ def get_qga_pkg_path(self, qemu_ga_pkg, test, session, params, vm):
         if self.gagent_src_type == "url":
             gagent_host_path = params["gagent_host_path"]
             gagent_download_url = params["gagent_download_url"]
+            mqgaw_ver = match = re.search(r'(?:\d+\.){2}\d+', gagent_download_url)
+            mqgaw_ver_list = list(map(int, mqgaw_ver.group(0).split('.')))
+            if mqgaw_ver_list >= [105, 0, 1]:
+                src_qgarpm_path = params["src_qgarpm_path_new"]
+                cmd_get_qgamsi_path = params["get_qgamsi_path_new"]
+            else:
+                src_qgarpm_path = params["src_qgarpm_path"]
+                cmd_get_qgamsi_path = params["get_qgamsi_path"]
             qga_msi = params['qemu_ga_pkg']
             rpm_install = "rpm_install" in gagent_download_url
             if rpm_install:
                 gagent_download_url = gagent_download_url.split("rpm_install:")[-1]
                 gagent_download_cmd = 'wget -qP %s %s' % (gagent_host_path,
                                                           gagent_download_url)
-                gagent_host_path += params["src_qgarpm_path"]
+                if mqgaw_ver_list >= [105, 0, 1]:
```

Comment:
I think there is no need to repeatedly obtain the value here.

Reply:
okay. trying it.